tests/aws/services/events/test_events_targets.py

```diff
@@ -37,6 +37,117 @@
 #   - Container (pro)
 #   - Redshift (pro)
 #   - Sagemaker (pro)
+class TestEventsTargetCloudWatchLogs:
+    @markers.aws.validated
+    def test_put_events_with_target_cloudwatch_logs(
+        self,
+        events_create_event_bus,
+        events_put_rule,
+        aws_client,
+        snapshot,
+        account_id,
+        region_name,
+    ):
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.key_value("EventId"),
+                snapshot.transform.key_value("RuleArn"),
+                snapshot.transform.key_value("EventBusArn"),
+            ]
+        )
+
+        # Step 1: Create an EventBridge event bus
+        event_bus_name = f"test-bus-{short_uid()}"
+        event_bus_response = events_create_event_bus(Name=event_bus_name)
+        snapshot.match("event_bus_response", event_bus_response)
+
+        # Step 2: Create a CloudWatch Logs log group
+        log_group_name = f"/aws/events/test-log-group-{short_uid()}"
+        aws_client.logs.create_log_group(logGroupName=log_group_name)
+        log_group_arn = f"arn:aws:logs:{region_name}:{account_id}:log-group:{log_group_name}"
+
+        # Step 3: Attach a resource policy to the log group to allow EventBridge to write to it
+        resource_policy = {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "EventBridgePutLogEvents",
+                    "Effect": "Allow",
+                    "Principal": {"Service": "events.amazonaws.com"},
+                    "Action": ["logs:CreateLogStream", "logs:PutLogEvents"],
+                    "Resource": f"{log_group_arn}:*",
+                }
+            ],
+        }
+        policy_name = f"EventBridgePolicy-{short_uid()}"
+        aws_client.logs.put_resource_policy(
+            policyName=policy_name, policyDocument=json.dumps(resource_policy)
+        )
+
+        # Wait for the resource policy to take effect (only needed in AWS)
+        if is_aws_cloud():
+            time.sleep(10)
+
+        # Step 4: Create a rule on the event bus
+        rule_name = f"test-rule-{short_uid()}"
+        event_pattern = {"source": ["test.source"], "detail-type": ["test.detail.type"]}
```

Comment:
you can use the default pattern TEST_EVENT_PATTERN from test_events - makes it easier to avoid problems with not matching rules - and also use the default message from test_events